src/vm/amd64/cgencpu.h

```diff
@@ -215,6 +195,16 @@ void UpdateRegDisplayFromCalleeSavedRegisters(REGDISPLAY * pRD, CalleeSavedRegis
 
 #define NUM_ARGUMENT_REGISTERS 6
 
+#define ENUM_CALLEE_SAVED_REGISTERS() \
+    CALLEE_SAVED_REGISTER(R12) \
+    CALLEE_SAVED_REGISTER(R13) \
+    CALLEE_SAVED_REGISTER(R14) \
+    CALLEE_SAVED_REGISTER(R15) \
+    CALLEE_SAVED_REGISTER(Rbx) \
+    CALLEE_SAVED_REGISTER(Rbp)
+
+#define NUM_CALLEE_SAVED_REGISTERS 8
```

Comment:
Good catch! The memory corruptions from this typo would not be fun to trace down.